### PR TITLE
Roc 4571

### PR DESCRIPTION
**Pull request by karthikkannankainos**
### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: src/test/features/claimant-response/helpers/settlementHelper.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ describe('settlementHelper', () => {
       expect(partyStatement.madeBy).to.be.eql('DEFENDANT')
       expect(partyStatement.type).to.be.eql('OFFER')
       expect(partyStatement.offer).is.not.undefined
-      expect(partyStatement.offer.content).to.be.eql('John Smith will pay the full amount, no later than 31 December 2050')
+      expect(partyStatement.offer.content).to.be.eql('John Smith will pay £200, no later than 31 December 2050')
```
Comment (pragnesh-patel) [inline src/test/features/claimant-response/helpers/settlementHelper.ts]: hardcoded ? take from test data?

Comment (rattur) [reply src/test/features/claimant-response/helpers/settlementHelper.ts]: Can this review comment be addressed so that I can approve PR?

Comment (karthikkannankainos) [reply src/test/features/claimant-response/helpers/settlementHelper.ts]: @pragnesh-patel the value is derived and it cant be retrieved from the test data. So only we can check this using hard coded value.

Comment (srineelesh) [inline src/test/features/claimant-response/helpers/settlementHelper.ts]: There is a test case .it('should return defendant party statement by installment option'. I see that the assertion for the offer content is missing. Can you add that as it related to these changes

Comment (karthikkannankainos) [reply src/test/features/claimant-response/helpers/settlementHelper.ts]: @srineelesh good point. Have fixed it.

**Comment by karthikkannankainos** [comment]
The commit tree in this PR got spoilt after rebasing master. So raised a separate PR with just one commit. Thus closing this.

**Pull request closed without merging**